src/libraries/System.Threading.ThreadPool.WebAssembly.Threading/Directory.Build.props

```diff
@@ -0,0 +1,9 @@
+<Project>
+  <Import Project="..\Directory.Build.props" />
+  <PropertyGroup>
+    <!-- Disable all binplacing since this assembly will clash with the real System.Runtime -->
+    <EnableBinPlacing>false</EnableBinPlacing>
+    <StrongNameKeyId>Microsoft</StrongNameKeyId>
+    <IncludePlatformAttributes>true</IncludePlatformAttributes>
```

Comment:
Directory.Build.props files only make sense if the directory contains multiple projects (that compile). I would delete all these files and move these three properties directly into the ref projects.